modules/apps/object/object-test/src/testFunctional/tests/ObjectPersonalData.testcase

```diff
@@ -22,6 +22,273 @@ definition {
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject186468");
+
+			for (var entryName : list "B,C,D,E") {
+				ObjectAdmin.goToCustomObject(objectName = "CustomObject186468");
+
+				ObjectPortlet.addSingleFieldEntryViaUI(entry = "Entry ${entryName}");
+			}
+		}
+
+		task ("When the admin user anonymizes the user's personal data in Users and Organizations") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "test@liferay.com",
+				userLoginFullName = "Test Test");
+
+			ApplicationsMenu.gotoPortlet(
+				category = "Users",
+				panel = "Control Panel",
+				portlet = "Users and Organizations");
+
+			User.deletePersonalDataCP(
+				userFirstName = "userfn",
+				userLastName = "userln");
+
+			GDPR.anonymizeAllEntriesinApplicationCP(
+				applicationItemsCount = 4,
+				applications = "Objects",
+				totalItemsCount = 4);
+		}
+
+		task ("Then the author of the entries created by the user is Anonymous Anonymous") {
+			User.openUsersAdmin();
+
+			User.activateCP(
+				userName = "userfn userln",
+				userScreenName = "usersn");
+
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "userea@liferay.com",
+				userLoginFullName = "userfn userln");
+
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject186468");
+
+			for (var key_order : list "2,3,4,5") {
+				AssertElementPresent(
+					key_entry = "Anonymous Anonymous",
+					key_order = ${key_order},
+					locator1 = "ObjectPortlet#ENTRY_VALUE_BY_POSITION");
+			}
+		}
+	}
+
+<<<<<<< HEAD
+=======
+	@description = "LPS-186467 - Verify that the entries of a object can be deleted"
+	@priority = 5
+	test CanDeleteObjectEntries {
+		property portal.acceptance = "true";
+
+		task ("Given a custom object in a M:M relationship with itself") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 186467",
+				objectName = "CustomObject186467",
+				pluralLabelName = "Custom Objects 186467");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Custom Field",
+				fieldName = "customObjectField",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "CustomObject186467");
+
+			ObjectAdmin.addObjectRelationshipViaAPI(
+				objectName_1 = "CustomObject186467",
+				objectName_2 = "CustomObject186467",
+				relationshipLabel = "Relationship",
+				relationshipName = "relationship",
+				relationshipType = "manyToMany");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject186467");
+		}
+
+		task ("And given a custom layout that contains the relationship") {
+			ObjectAdmin.addObjectLayoutViaAPI(
+				layoutName = "Layout Name",
+				objectName = "CustomObject186467");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 186467");
+
+			CreateObject.selectTitleField(fieldLabel = "Custom Field");
+
+			CreateObject.saveObject();
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 186467");
```

Comment:
Do we need to navigate back to the object admin page here? Why not go straight to the layouts tab?